language-plutus-core/generators/Language/PlutusCore/Generators/Interesting.hs

```diff
@@ -1 +1,97 @@
-module Language.PlutusCore.Generators.Interesting where
+module Language.PlutusCore.Generators.Interesting
```

Comment:
This is still a somewhat obscure module name, so it could at least do with some module Haddock explaining what it's for.

Reply:
I do not like the name as well. Do you have a better suggestion? `Sample` maybe? These are just random term generators which I consider interesting enough to have.

Reply:
I think it's okay if there's an explanation attached. You could even go as far as calling it something derived from "evaluator test support/test cases".